### PR TITLE
Do not feel obligated to approve

### PR DESCRIPTION
**Pull request by bsdpunk**
I just moved the support rb files to lib/ and spruced up the readme. Feel free to reject.

### CODE REVIEW

File: war.rb [hunk 1]
```diff
@@ -74,4 +74,4 @@
   puts "-------------------------"
   puts "----Player 2 WINS!!!!----"
   puts "-------------------------"
-end
\ No newline at end of file
+end
```
Comment (Brandongoodman615) [inline war.rb:77]: Was this not there already?

File: README.md [hunk 2]
```diff
@@ -1,3 +1,6 @@
-# war_game
-#
-# $ ruby war.rb
+# War, The Card Game
+```
+ $ ruby war.rb
+```
+
+![Example Play](ss.png)
```
Comment (Brandongoodman615) [inline README.md:6]: What does this do?

File: war.rb (hunk 1 above)
Comment (bsdpunk) [reply war.rb:77]: The Readme was there. I put 
"$ ruby war.rb"
In markdown, with syntax, ie
```
$ruby war.rb
```

File: README.md (hunk 2 above)
Comment (bsdpunk) [reply README.md:6]: I uploaded a picture with Gameplay, and then added that picture to the README.

File: war.rb (hunk 1 above)
Comment (Brandongoodman615) [reply war.rb:77]: The PR shows this end was added, but I had it there already. I might just be looking at it wrong.

**Pull request merged**